File: app/src/test/java/com/mgaetan89/showsrage/databinding/Setters_SetSelectedTest.java
package com.mgaetan89.showsrage.databinding;

import android.widget.TextView;

import org.junit.Test;

import static org.mockito.Mockito.mock;
import static org.mockito.Mockito.verify;

public class Setters_SetSelectedTest {
	@Test
	public void setSelectedFalse() {
		TextView textView = mock(TextView.class);

		Setters.setSelected(textView, false);

		verify(textView).setSelected(false);
	}

	@Test
	public void setSelectedTrue() {
		TextView textView = mock(TextView.class);

		Setters.setSelected(textView, true);

		verify(textView).setSelected(true);
	}
}

File: app/src/test/kotlin/com/mgaetan89/showsrage/databinding/Setters_SetSelectedTest.kt
package com.mgaetan89.showsrage.databinding

import android.widget.TextView

import org.junit.Test

import org.mockito.Mockito.mock
import org.mockito.Mockito.verify

class Setters_SetSelectedTest {
	@Test
	fun setSelectedFalse() {
		val textView = mock(TextView::class.java)

		setSelected(textView, false)

		verify(textView).isSelected = false
	}

	@Test
	fun setSelectedTrue() {
		val textView = mock(TextView::class.java)

		setSelected(textView, true)

		verify(textView).isSelected = true
	}
}
